content/kubermatic/master/tutorials/13-psp/_index.en.md

```diff
@@ -26,6 +28,10 @@ For existing clusters, it's also possible to enable/disable PSP:
 Activating Pod Security Policy will mean that a lot of Pod specifications, Operators and Helm charts will not work out of the box. Kubermatic will apply a default authorizing policy to prevent this. Additionally, all Kubermatic user-clusters are configured to be compatible with enabled PSPs. Make sure that you know the consequences of activating this feature on your workloads.
 {{% /notice %}}
 
+### Datacenter Level Support
+It is also possible to enforce enabling Pod Security Policies on the datacenter level. In this case, user cluster level configuration will be ignored, and PSP will be enabled for all user clusters in the datacenter.
+
+To enable this, you will need to edit your [datacenters.yaml]({{< ref "../../concepts/datacenters/" >}}) or your [Seed Cluster CRD]({{< ref "../../concepts/seeds/" >}}), and set `enforcePodSecurityPolicy` to `true` in the datacenter spec.
```

Comment:
Can we rephrase this as

> you will need to edit your Seed CRD (or your datacenters.yaml in legacy Kubermatic versions) and set ....

In my perfect world, I would not mention the old datacenters.yaml at all anymore, but people seem to like having docs for it even in the master version. I still would at least mention the modern, new way *first* :)